src/generated/resources/assets/dungeonsdelight/lang/zh_cn.json

```
@@ -0,0 +1,402 @@
+{
+  "block.dungeonsdelight.cut_stained_scrap": "切制污秽残片",
+  "block.dungeonsdelight.cut_stained_scrap_slab": "切制污秽残片台阶",
+  "block.dungeonsdelight.cut_stained_scrap_stairs": "切制污秽残片楼梯",
+  "block.dungeonsdelight.dungeon_stove": "地牢炉灶",
+  "block.dungeonsdelight.embedded_eggs": "嵌入蛋",
+  "block.dungeonsdelight.glow_berry_gelatin_block": "发光浆果凝胶块",
+  "block.dungeonsdelight.guardian_angel_block": "守护天使",
+  "block.dungeonsdelight.heap_of_ancient_eggs": "远古蛋堆",
+  "block.dungeonsdelight.living_fire": "活焰",
+  "block.dungeonsdelight.monster_cake": "怪物蛋糕",
+  "block.dungeonsdelight.monster_pot": "怪物锅",
+  "block.dungeonsdelight.ossobusco_block": "红烩人小膝",
+  "block.dungeonsdelight.poisonous_potato_crate": "箱装毒马铃薯",
+  "block.dungeonsdelight.rotbulb_crate": "箱装腐球茎",
+  "block.dungeonsdelight.rotbulb_crop": "腐球茎作物",
+  "block.dungeonsdelight.rotbulb_plant": "野生腐球茎",
+  "block.dungeonsdelight.rotten_crop": "腐烂作物",
+  "block.dungeonsdelight.rotten_potatoes": "腐烂马铃薯",
+  "block.dungeonsdelight.rotten_tomato_crate": "箱装烂番茄",
+  "block.dungeonsdelight.rotten_tomatoes": "箱装烂番茄",
+  "block.dungeonsdelight.sculk_mayo_block": "幽匿蛋黄酱",
+  "block.dungeonsdelight.sculk_tart": "幽匿馅饼",
+  "block.dungeonsdelight.spider_pie": "蜘蛛派",
+  "block.dungeonsdelight.stained_scrap_bars": "污秽残片条",
```

```suggestion
  "block.dungeonsdelight.stained_scrap_bars": "污秽残片栏杆",
```
这是对应铁栏杆（Iron Bars）的建筑方块，请对照[合成表](https://github.com/Yirmiri/Dungeons-Delight/blob/1.20.x/src/generated/resources/data/dungeonsdelight/recipes/stained_scrap_bars.json)和游戏内表现翻译。